### PR TITLE
./misc/Makefile: Use of variables

### PR DESCRIPTION
**Pull request by SubbulakshmiRS**
Variables like $(mandir),$(prefix),$(datadir) and $(datarootdir)
 are used in the Makefile .

Closes https://github.com/qutebrowser/qutebrowser/issues/3538

---
This change is [Reviewable](https://reviewable.io/reviews/qutebrowser/qutebrowser/4107)

### CODE REVIEW

**Comment by SubbulakshmiRS** [comment]
should i also use "?=" for mandir and datarootdir variables ?

File: misc/Makefile [hunk 1]
```diff
@@ -1,7 +1,10 @@
 PYTHON = python3
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 DESTDIR =
 ICONSIZES = 16 24 32 48 64 128 256 512
+DATAROOTDIR = $(PREFIX)/share
+DATADIR ?= $(DATAROOTDIR)
+MANDIR = $(DATAROOTDIR)/man
```
[the remark this answers is not in the document]
Comment (SubbulakshmiRS) [reply misc/Makefile]: made the changes to mandir

**Comment by The-Compiler** [comment]
@glatzor @gebulmer If you have the time, can you take a quick look at this please?

**Comment by The-Compiler** [comment]
Thanks, and sorry for the delay!

**Pull request merged**